tests/gitlab/jspaaks/badge_test/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .mocker import mocker
+
+
+__all__ = [
+    "mocker"
+]
```

Comment:
Can change the import in the testcases to `from .mocker import mocker` so we don't need this export.
```suggestion
```

Reply:
done